tests/ast/data/session_write_pandas.test

```diff
@@ -0,0 +1,15 @@
+## TEST CASE
+
+import pandas as pd
+
+df = pd.DataFrame([[1, 2, 3], [4, 5, 6]])
+
+ans = session.write_pandas(df, "test_table")
+
+ans2 = session.write_pandas(df, "test", schema="a", database="b", chunk_size=7, compression="brotli", on_error="ignore", parallel=10, quote_identifiers=False, auto_create_table=True, create_temp_table=True, overwrite=True, table_type="temp", random=90)
+
+## EXPECTED OUTPUT
+
+res1 = session.write_pandas(pandas.DataFrame(<not shown>)), "test_table")
```

Comment:
Strange – why is the symbol not getting captured here?

Reply:
Good catch! Looked into it, in the `with_src_position` statement I forgot the `stmt` as parameter. Fixed now, i.e.

```
with_src_position(stmt.expr.sp_write_pandas, stmt)
```